tensorflow_asr/models/keras/transducer.py

```diff
@@ -47,14 +46,17 @@ def call(self, inputs, training=False, **kwargs):
             "logit_length": get_reduced_length(inputs["input_length"], self.time_reduction_factor)
         }
 
-    def train_step(self, data):
-        data = data_adapter.expand_1d(data)
-        x, y, sample_weight = data_adapter.unpack_x_y_sample_weight(data)
-
+    def train_step(self, batch):
+        x, y_true = batch
         with tf.GradientTape() as tape:
             y_pred = self(x, training=True)
-            loss = self.loss(y, y_pred)
+            loss = self.loss(y_true, y_pred)
```

Comment:
Pls adding Mixed precision support here

Reply:
loss scaling added, in fact, it doesn't need loss scaling because the loss is already too big at the beginning but I added anyway